lib/wsdl.js

```diff
@@ -1957,5 +1975,13 @@ function open_wsdl(uri, options, callback) {
   return wsdl;
 }
 
+function clear_cache() {
+  WSDL_CACHE = {};
```

Comment:
It's generally better to clear object references to avoid bugs down the road.  I've been bitten by this many times, so it's more about experience rather than a feeling.